requirements.txt

```diff
@@ -84,7 +84,7 @@ task-processing==1.3.5
 traitlets==5.0.0
 Twisted==22.10.0
 typing-extensions==4.5.0
-urllib3==1.26.5
+urllib3==1.25.10
```

Comment:
it probably wouldn't be too hard to update `requests`, but we can also punt on this :p